pkg/analytics/track.go

```diff
@@ -333,6 +340,7 @@ func track(event string, success bool, props map[string]interface{}) {
 	props["success"] = success
 	props["contextType"] = getContextType(okteto.Context().Name)
 	props["context"] = okteto.Context().Name
+	props["cluster"] = okteto.Context().Name
```

Comment:
isn't this the same as `context`?

Reply:
Yes, I was keeping that for retrocompatibility, but we can remove it if we don't need it `context` anymore

Reply:
wouldn't it be better to use `context` prop instead?

Reply:
I think this is due to the fact that the backend is using `cluster` property instead of `context`. We need to have the same props on both so this is why we changed here, but I don't know if it makes more sense to change it in the backend to `context`

Reply:
`cluster` is what was defined in the spec, and what has been implemented in other events in both backend and frontend. These changes try to unify the CLI with the way we do it in other places.

Reply:
It's true that with the new context command it would've been better to use `context`, but for now I think it ok to use "cluster" and have the same property across all services (frontend, backend and cli).

Reply:
@rlamana let's have both then? they seem synonymous to me, and the right concept in the okteto platform is `context`. But having both props doesn't hurt if it simplifies the e2e configuration

Reply:
I'm ok with having both.